src/main/java/net/snowflake/client/jdbc/SnowflakeConnectionV1.java

```diff
@@ -359,21 +358,21 @@ public CallableStatement prepareCall(
 
   @Override
   public String nativeSQL(String sql) throws SQLException {
-    logger.debug("public String nativeSQL(String sql)", false);
+    logger.trace("public String nativeSQL(String sql)", false);
     raiseSQLExceptionIfConnectionIsClosed();
     return sql;
   }
 
   @Override
   public boolean getAutoCommit() throws SQLException {
-    logger.debug("boolean getAutoCommit()", false);
+    logger.trace("boolean getAutoCommit()", false);
```

Comment:
in other logs we have public at the beginning - we should make it the same (personally opt to remove public from other logs)

Reply:
Done